content/en/docs/reference/scheduling/config.md

```diff
@@ -255,10 +254,47 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
+## Scheduler configuration migrations
+
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1beta2 → v1beta3" %}}
+* With the v1beta3 configuration version, you will have increased weights for the following user specifiable scoring plugins.
+  - TaintTolerations to 3
+  - NodeAffinity to 2
+  - InterPodAffinity to 2
+  - Won't have HealthzBindAddress, MetricsBindAddress fields 
+
+* With the v1beta2 configuration version, you can use a new score extension for the
```

Comment:
why is this listed in the v1beta3 migration?

Reply:
That's why I mentioned this need to be change. https://github.com/kubernetes/website/pull/30441/files#diff-4a9a9ec0b9a1d8e703de27bf418c577795b0b8410f37cd17d399133b267af1adR268